stellargraph/core/graph.py

```diff
@@ -279,52 +283,75 @@ def from_networkx(
 
         Args:
             graph: The NetworkX graph instance.
-            node_type_name: str, optional (default=globals.TYPE_ATTR_NAME)
+            node_type_attr (str, optional):
                 This is the name for the node types that StellarGraph uses
                 when processing heterogeneous graphs. StellarGraph will
                 look for this attribute in the nodes of the graph to determine
                 their type.
 
-            node_type_default: str, optional (default=globals.NODE_TYPE_DEFAULT)
+            node_type_default (str, optional):
                 This is the default node type to use for nodes that do not have
                 an explicit type.
 
-            edge_type_name: str, optional (default=globals.TYPE_ATTR_NAME)
+            edge_type_attr (str, optional):
                 This is the name for the edge types that StellarGraph uses
                 when processing heterogeneous graphs. StellarGraph will
                 look for this attribute in the edges of the graph to determine
                 their type.
 
-            edge_type_default: str, optional (default=globals.EDGE_TYPE_DEFAULT)
+            edge_type_default (str, optional):
                 This is the default edge type to use for edges that do not have
                 an explicit type.
 
-            node_features: str, dict, list or DataFrame optional (default=None)
+            node_features (str, dict, list or DataFrame optional):
                 This tells StellarGraph where to find the node feature information
                 required by some graph models. These are expected to be
                 a numeric feature vector for each node in the graph.
 
-            edge_weight_label: str, optional
+            edge_weight_attr (str, optional):
                 The name of the attribute to use as the weight of edges.
 
+            node_type_name: Deprecated, use ``node_type_attr``.
+            edge_type_name: Deprecated, use ``edge_type_attr``.
+            edge_weight_label: Deprecated, use ``edge_weight_attr``.
+
         Returns:
             A ``StellarGraph`` (if ``graph`` is undirected) or ``StellarDiGraph`` (if ``graph`` is
             directed) instance representing the data in ``graph`` and ``node_features``.
         """
+        if node_type_name is not None:
+            warnings.warn(
+                "the 'node_type_name' parameter is has been replaced by 'node_type_attr'",
```

Comment:
```suggestion
                "the 'node_type_name' parameter has been replaced by 'node_type_attr'",
```
and similarly for the two below